magprime/automated_emails.py

```diff
@@ -57,7 +57,6 @@ def __init__(self, event):
     Attendee, 'Department Heads', 'food/department_heads.txt',
     lambda a: a.is_dept_head,
     ident='magprime_department_water_and_food_info',
-   # when=days_before(7, c.UBER_TAKEDOWN),
```

Comment:
There's no reason to have this commented-out line and it was a linter error anyway.